Atomic/syscontainers.py

```diff
@@ -174,7 +174,11 @@ def _generate_default_oci_configuration(self, destination):
         with open(conf_path, 'r') as conf:
             configuration = json.loads(conf.read())
         configuration['root']['readonly'] = True
-        configuration['root']['path'] = 'rootfs'
+        if self.args.remote:
+            configuration['root']['path'] = self.args.remote + "/rootfs"
```

Comment:
the `_generate_default_oci_configuration` function is used only for the images that have not a configuration file under the `/exports` directory.

We need to set ``configuration['root']['path']`` after the configuration file has been generated, either the default one as covered here, or if it is copied directly or through the template file.

My suggestion is that this is done just before `_check_oci_configuration_file` is called.

Later we will need to refactor this part though, as the file is opened multiple times, but this can be done out of this patch.